fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -619,6 +628,13 @@ public Response createObject(@HeaderParam(CONTENT_DISPOSITION) final ContentDisp
         }
     }
 
+    private void checkAclUriExists(final URI resourceAcl) {
+        final FedoraResource aclResource = getResourceFromPath(resourceAcl.getPath());
```

Comment:
I think you'll need a try catch here that checks if exception itself or its cause is an instance of PathNotFoundRuntimeException.